pkg/util/error.go

```diff
@@ -40,3 +40,16 @@ func CheckErr(err error, handleErr func(string)) {
 	}
 	handleErr(err.Error())
 }
+
+// Handle Empty Table Error
```

Comment:
```suggestion
// HandleEmptyTableError handles errors when a resource or set of resources aren't found
```

This is a minor linting issue in golang, exported functions must be properly documented as such.
Ref: https://go.dev/blog/godoc

Reply:
Sounds great, Done